packages/odyssey-react/src/components/Toast/index.tsx

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { createContext, useContext, useState } from "react";
+import type { FunctionComponent, ReactElement, AnimationEvent } from "react";
+import { useCx, useOid, oid } from '../../utils';
+import Button from '../Button';
+
+export type ToastVariants = 'info' | 'success' | 'caution' | 'danger';
+
+export interface PropsToast {
+  /**
+   * The title to be displayed on the toast. Provides quick context; one line max.
+   */
+  title: string,
+  
+  /**
+   * Supplemental information. Be concise - less than three lines of content - as your Toast will soon vanish!
+   */
+  body?: string,
+
+ /**
+   * The visual variant to be displayed to the user.
+   * @default info
+   */
+  variant?: ToastVariants,
+  
+ /**
+   * The visual variant to be displayed to the user.
+   * @default info
+   */
+  id?: string,
+  
+  onAnimationEnd?: (event: AnimationEvent) => void,
+
+  onDismiss: () => void
+}
+
+export type PropsToastProvider = { 
+  /**
+   * Child react nodes which leverage the toast context. This is typically an entire app. 
+   */
+  children?: ReactElement | ReactElement[],
```

Comment:
if we want to be flexible in the content that can be rendered within a `Toast.Provider` why restrict it to `ReactElements` here? I would think we'd want `ReactNode` yea?

Reply:
Good catch, has been fixed.